common/protob/messages-debug.proto

```diff
@@ -54,6 +54,7 @@ message DebugLinkState {
     optional uint32 recovery_word_pos = 10;                 // index of mnemonic word the device is expecting during RecoveryDevice workflow
     optional uint32 reset_word_pos = 11;                    // index of mnemonic word the device is expecting during ResetDevice workflow
     optional uint32 mnemonic_type = 12;                     // current mnemonic type (BIP-39/SLIP-39)
+    optional bool sd_card = 13;                             // is sd card present
```

Comment:
Personally I would prefer to call this `sd_card_present`, but I don't insist.

Reply:
I think the bool implies it. Let's keep it and rename in case we need more sd card fields.

Reply:
My take is it's not a secret and we can report that in Features.

Reply:
It is not secret. I was doing this under the assumption that we are trying to have Features thin and add things there only when necessary (which might be that case very soon).

Reply:
i was under the impression that we can't really make Features thinner, and the idea is instead to not send Features every time?